examples/point-of-sale/src/client/components/buttons/BackButton.module.css

```diff
@@ -1,5 +1,5 @@
 .button {
-    height: 48px;
+    height: 40px;
```

Comment:
48px is actually the Apple Human Interface Guideline for the size of tap targets. You can make the size of the icon smaller, but keep the tap target 48px.

Reply:
https://web.dev/accessible-tap-targets/

Reply:
Thanks for the teaching!
I'll pay attention to that from now on...
Testing with a small device like IPhone SE to handle the case where the Generate button is not visible anymore.
Changing the margin / padding does not change the layout but keep the button size :-)